doc/01 Route Configuration/Plain Route.md

```diff
@@ -48,10 +48,12 @@ var myRoute = {
   path: 'picture/:id',
   getChildRoutes (state, cb) {
     // state gets passed to `getChildRoutes`
-    if (state && state.fromDashboard)
+    if (state && state.fromDashboard) {
       cb(null, [dashboardPictureRoute])
-    else
+    }
+    else {
```

Comment:
Yeah, that's mostly a personal preference, honestly. I feel like having `else` on the same line as the closing curly brace for `if` kind of makes `else` feel like a 'second-class citizen', even though its logic might be just as (or even more) important.

I realize I'm most likely in the minority regarding this, so if that's something that should be changed, I'd be happy to.